server/models/users_test.go

```diff
@@ -0,0 +1,93 @@
+package models
+
+import (
+	"context"
+	"log"
+	"testing"
+
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/jmoiron/sqlx"
+	pb "github.com/naoyakurokawa/app-grpc-web/hello"
+	"golang.org/x/crypto/bcrypt"
+)
+
+const (
+	Name     = "テスト"
+	Score    = 123
+	Photourl = "https://test"
+	Password = "abcd12341231"
+)
+
+// ユーザ取得 正常
+func TestGetUsersSuccess(t *testing.T) {
+	db, err := sqlx.Open("mysql", "root:test@tcp(127.0.0.1:13306)/test")
+	if err != nil {
+		log.Fatalln(err)
+	}
+	ctx := context.Background()
+	InitUserTable(db)
+	CreateUserForTest(db)
+	request := pb.GetUsersRequest{}
+	users, err := GetUsers(ctx, db, request)
+	if err != nil {
+		t.Error("\n実際： ", "エラー", "\n理想： ", "正常終了")
+	}
+	if users[0].Name != "テスト" {
+		t.Errorf("%v != %v", users[0].Name, "テスト")
+	}
+	if users[0].Score != 123 {
+		t.Errorf("%v != %v", users[0].Score, 123)
+	}
+	if users[0].Photourl != "https://test" {
+		t.Errorf("%v != %v", users[0].Photourl, "https://test")
+	}
+}
+
+func InitUserTable(db *sqlx.DB) error {
```

Comment:
パッケージ内でしか使用されないメソッドですので、
（関数名の先頭を小文字にして）プライベートにしておきましょう。
```suggestion
func initUserTable(db *sqlx.DB) error {
```